client/client.go

```diff
@@ -263,7 +266,10 @@ func (c *Client) GetJSONWebKeys() *jose.JSONWebKeySet {
 }
 
 func (c *Client) GetTokenEndpointAuthSigningAlgorithm() string {
```

Comment:
This new functionality needs a test: https://github.com/ory/hydra/blob/f0609ad96d8393ad96b670314faa0cff0ea39605/client/manager_test_helpers.go#L71

It also needs validation (and a test for validation):

https://github.com/ory/hydra/blob/master/client/validator.go#L116